### PR TITLE
Updates suggested during approval of RFC-708.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

File: LDM-148.tex [hunk 1]
```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
...
+\subsection{Data Flow}\label{data-flow}
+
+Images (science and calibration) are taken on the mountain and stored in the camera image buffer (sized for a week or more).
+Images are copied by the camera software to the Camera Diagnostic Cluster for rapid analysis.
+Images may reside on that cluster for a short time.
+Images are also transferred to the Observatory Operations Data Service (OODS) for use by the OCS-Controlled Pipeline Service (OCPS), the Commissioning Cluster, and other Observatory systems such as the Active Optics System.
+They are expected to reside in the OODS for at least 30 days.
+At the time of readout, images are sent quickly to the US Data Facility (USDF) for Prompt Processing.
+A copy of the raw images is kept in the Base Facility in Chile as part of the Data Backbone, and an archival copy is also maintained in the Data Backbone at the US Data Facility.
+After an agreed delay period, the raw images are made available to the data rights holders in both the US and Chilean Data Access Centers (DACs) via the LSST Science Platform.
```
Comment (rblum5) [inline LDM-148.tex:170]: Suggest "After an agreed upon delay period with NSF, DOE, SLAC, and AURA, the raw images are made available to all approved data rights holders. Access is at least  from both the US and Chilean Data Access Centers (DACs) via the LSST Science Platform but could also include other Independent Data Access Centers (IDAC)."

File: LDM-148.tex [hunk 2]
```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
...
+During the 24 hours after readout, raw images are sent to the Satellite Data Facility at CC-IN2P3 both as a disaster recovery backup and to facilitate the Data Release Processing there at a later date.
```
Comment (rblum5) [inline LDM-148.tex:171]: Suggest: "During the 24 hours after readout, raw images are sent to the French Data Facility (FrDF) at CC-IN2P3 ..."

Comment (rblum5) [reply LDM-148.tex:171]: "... at a later date. Processing may also be done at other international data facilities."

File: LDM-148.tex [hunk 3]
```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
...
+
+Information about the state of the Observatory (parts of which are sometimes referred to as image metadata) is collected from messages in the Observatory control network and transmitted via two mechanisms.
+The Header Service component of the Archiving service collects information required for the OCPS and Prompt Processing and creates FITS headers that are attached to the image pixels during export from the camera image buffer.
+Separately, the Engineering and Facility Database (EFD) collects all control messages and transports them via Kafka messaging to a replica at the US Data Facility, where they are aggregated and summarized for eventual use by the Calibration Products Production.
+The EFD data is made available to the US and Chilean DACs as well.
+
+Alerts generated by the Alert Production payload in the Prompt Processing service are distributed as rapidly as possible by the Alert Distribution service.
```
Comment (rblum5) [inline LDM-148.tex:178]: Suggest: "Alerts generated by the Alert Production payload in the Prompt Processing service are distributed as rapidly as possible by the Alert Distribution service. This is currently envisioned to be 60s after the image is taken in Chile."

File: LDM-148.tex [hunk 4]
```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
...
+
+Other data products, whether images or catalogs and whether Prompt or Data Release, are generated within the USDF or the Satellite Data Facility.
```
Comment (rblum5) [inline LDM-148.tex:180]: Suggest " ... generated within the USDF or the FrDF."

File: LDM-148.tex [hunk 5]
```diff
@@ -158,6 +158,31 @@ \section{Summary Concept of Operations}\label{summary-concept-of-operations}
...
+They are archived in the Data Backbone at the USDF which then replicates them to the Base Facility in Chile.
+Staff have access to these data products through the LSST Science Platform Science Validation instance.
```
Comment (rblum5) [inline LDM-148.tex:182]: Should be Rubin Science Platform.

Comment (ktlim) [reply LDM-148.tex:183]: Not yet; we agreed for the JSR to leave everything as LSP.

**Pull request merged**